docs/ff-concepts/adding-customization/cloud-functions.md

```diff
@@ -245,6 +245,45 @@ For this example, we'll use the result (i.e., generated logo image URL) and set
 <div class="video-container"><iframe src="https://www.loom.
 com/embed/0c4306c1951a4d9099aa96324c7561af?sid=69709110-ad60-4e98-bf53-36a50a99e425" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></div>
 
+## Testing Cloud Functions
+
+### How can I test sending push notifications without writing to Firestore?
```

Comment:
Another header won't be required. So we can remove this.

Reply:
This title is more content-bearing than the one above. If we want to remove either, I'd probably remove the "Testing Cloud Functions" one.

I'll update this one to be in the correct format (it's a question, because I initially wrote this as an FAQ item).